recipes/rust-activation/forge_test.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -e -x
+
+## TODO: remove the following `unset` lines, once the following issue in `conda-build` is resolved:
+##       <https://github.com/conda/conda-build/issues/2255>
+
+unset REQUESTS_CA_BUNDLE
+unset SSL_CERT_FILE
+
+rustc --help
+rustdoc --help
+cargo --help
+
+echo "#!/usr/bin/env bash"                         > ./cc
+if [[ ${target_platform} =~ linux.* ]]; then
+  echo "x86_64-conda_cos6-linux-gnu-cc \"\$@\""   >> ./cc
+elif [[ ${target_platform} == osx-64 ]]; then
+  echo "x86_64-apple-darwin13.4.0-clang \"\$@\""  >> ./cc
+  export CONDA_BUILD_SYSROOT=/opt/MacOSX10.10.sdk
```

Comment:
I guess this means rust needs 10.10 or later? If so, see: https://conda-forge.org/docs/maintainer/knowledge_base.html#requiring-newer-macos-sdks

Reply:
Good point.

We should see if we can setup a virtual package for the macOS version and I should figure out what rust 1.40.0 requires as the minimum runtime. The rust compiler activation package should then impose a strong run_export on the virtual package. All very possible I think, now. I should do some digging.